src/tree_store/page_store/mmap.rs

```diff
@@ -26,8 +26,18 @@ impl Mmap {
         self.inner.len()
     }
 
-    #[cfg(target_os = "macos")]
     pub(crate) fn flush(&self) -> Result {
+        self.inner.flush()?;
```

Comment:
On macos, this needs to call F_FULLFSYNC, doesn't it?

Reply:
Ah, right, fixed.